### ISSUE TITLE
Setup SAML integration with IdP metadata

### ISSUE THREAD

**Issue by malikkotob** [18F/beta.foia.gov#95]
As an agency user, in order to login to update agency and agency component contact information, I need the Drupal backend to authenticate with Max.gov.

## Acceptance criteria

- [x] Steps 3-7 from https://blt.readthedocs.io/en/8.x/readme/simplesamlphp-setup/#code-changes are completed as appropriate
- [x] Authentication with Max.gov is successful for user who exists in IdP using instructions from https://blt.readthedocs.io/en/8.x/readme/simplesamlphp-setup/#testing-library-configuration

## Developer Review

- [x] Review [PR #81](https://github.com/usdoj/foia-api/pull/81)
- [x] Navigate to http://foiadev.prod.acquia-sites.com/simplesaml/module.php/core/frontpage_config.php
- [x] Verify that the three links under the configuration section don't show any errors (note: you will have to login as an admin, contact Malik for the pw)
- [x] Follow functional review steps

## Functional Review

- [x] Follow the steps at https://test.max.gov/password to set a password in the max.gov test environment at https://test.max.gov/password
- [x] Navigate to http://foiadev.prod.acquia-sites.com/simplesaml/module.php/core/frontpage_auth.php
- [x] Click the 'Test configured authentication sources' link
- [x] Click the 'default-sp' link
- [x] Verify you are redirected to https://login.test.max.gov/
- [x] Enter your max.gov username and password you set in step 1 above
- [x] Click Login with User ID
- [x] Verify you are redirected to the test page on the foiadev environment
- [x] Verify you see a screen similar to the one Malik attached in a screenshot below with your specific max.gov info

**Comment by malikkotob** [comment]
This cannot be started until Acquia folks have access to the community page.

**Comment by malikkotob** [comment]
We have gained access to the community page, but are waiting for the IdP metadata to be able to start this work.

**Comment by malikkotob** [comment]
Successful authentication through `simplesaml` test page:
![image](https://user-images.githubusercontent.com/7820280/31177008-b3580f48-a8e2-11e7-9184-d5afba9819f3.png)

**Comment by malikkotob** [comment]
Blocked pending a reply from Rich at Max.gov.

**Comment by malikkotob** [comment]
Unblocked and ready for review.

**Comment by malikkotob** [comment]
Merged into `develop`.
